Exercise_D_E/index1.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Flower Shop</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./assets/style/index.css" />
+  </head>
+  <body>
+  
+    <div class= "header">
+      <img src="./assets/images/logo.png">
+    </div>
+  
+    <div class="flower-pic">
+      <img src="./assets/images/flowershop.jpg">
+      <div class="centered">Beautiful Flowers</div>
+      <button class= "learn-button"> Learn More</button>
+    </div>
+
+    <div class= "shop-features1">
+      <div class="flower-icon">
+       <img src="./assets/images/flower-icon.png">
+         <p class="flower">Flowers for every<br>kind of situation</p>
+      </div>
+      <div class= "payment-icon">
+       <img src="./assets/images/payment-icon.png">
+        <p>All major payments<br> kinds accepted</p>
+      </div>
+    </div>
+    <div class= "shop-features2">  
+      <div class= "phone-icon">
+       <img src="./assets/images/phone-icon.png">
+        <p>Free consultation</p>
+      </div>
+      <div class= "truck-icon">
+       <img src="./assets/images/truck-icon.png">
+        <p>City wide delivery</p>
+      </div>
+    </div>
+
+    <script src="" async defer></script>
+  </body>
+</html>
+
```

Comment:
Hey Alex 
Great work with this, this website has be designed according to the requirements